### PR TITLE
Add terminal-2 icon

### PR DESCRIPTION
**Pull request by mittalyashu**
| Preview |
| --- |
| CleanShot 2021-01-27 at 21 22 26@2x |

### CODE REVIEW

**Comment by locness3** [comment]
`terminal-square`?

**Comment by mittalyashu** [comment]
What about `terminal-app`?

**Comment by mittalyashu** [comment]
any updates on this PR?

**Review by ericfennis — APPROVED** [review]

**Review by ericfennis — CHANGES_REQUESTED** [review]
@mittalyashu Can you try to use the basic square shape?
image

File: icons/terminal-square.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M7 11l2-2-2-2" />
+  <path d="M11 13h4" />
+  <path d="M19 3H5a2 2 0 00-2 2v14a2 2 0 002 2h14a2 2 0 002-2V5a2 2 0 00-2-2z" />
```
Comment (ericfennis) [inline icons/terminal-square.svg:14]: shorter with rect element
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

**Pull request merged**